### PR TITLE
Create GinjFo.com.xml

### PR DESCRIPTION
**Pull request by 6u621jpq6ybdw**

### CODE REVIEW

File: src/chrome/content/GinjFo.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+	Invalid Certificate:
+		ginjfo.com
+		openx.ginjfo.com
```
Comment (J0WI) [inline src/chrome/content/GinjFo.com.xml]: Just an incomplete cert chain.

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

File: src/chrome/content/GinjFo.com.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/GinjFo.com.xml]: The comment was right for `^`

**Comment by koops76** [comment]
#11795

**Pull request merged**